File: Java/Intro/src/oop/AngajatiMain.java
package oop;

public class AngajatiMain {
    public static void main(String[] args) {
        Angajati daniel = new Angajati("Popescu","Daniel","Consilier",5500,12);
        Angajati georgel = new Angajati("Ionescu","Georgel","Referent",2800,3);

        georgel.descriereAngajat();
        georgel.actualizareVechime();
        georgel.marireSalariu();
        georgel.descriereAngajat();

        daniel.descriereAngajat();
        daniel.marireSalariu();
        daniel.descriereAngajat();
    }
}

File: Python/Intro/oop/angajati_main.py
from oop.angajati import Angajati

daniel = Angajati('Popescu','Daniel','Consilier',12,5500)
georgel = Angajati("Ionescu",'Georgel','Referent',3,2800)

daniel.descriereAngajat()
georgel.descriereAngajat()

georgel.actualizareVechime()
georgel.descriereAngajat()

daniel.marireSalariu()
daniel.descriereAngajat()
